framework/auth/core.py

```diff
@@ -657,9 +657,29 @@ def get_claim_url(self, project_id, external=False):
         return '{base_url}user/{uid}/{project_id}/claim/?token={token}'\
                     .format(**locals())
 
-    def set_password(self, raw_password):
-        """Set the password for this user to the hash of ``raw_password``."""
+    def set_password(self, raw_password, suppress_notification=False):
```

Comment:
Minor: parameter names that result in double negatives are confusing. How about `notify=True` instead?